dropwizard-scala-parent-pom/pom.xml

```diff
@@ -0,0 +1,218 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>com.mainstreethub</groupId>
+    <artifactId>dropwizard-parent-pom</artifactId>
+    <version>1.0.2-4</version>
+  </parent>
+
+  <artifactId>dropwizard-scala-parent-pom</artifactId>
+  <version>1.0.2-1-SNAPSHOT</version>
+  <packaging>pom</packaging>
+
+  <scm>
+    <url>https://github.com/mainstreethub/parent-poms</url>
+    <connection>scm:git:git@github.com:mainstreethub/parent-poms</connection>
+    <developerConnection>scm:git:git@github.com:mainstreethub/parent-poms</developerConnection>
+    <tag>HEAD</tag>
+  </scm>
+
+  <properties>
+    <dropwizard-scala.commons.version>0.0.1</dropwizard-scala.commons.version>
+    <dropwizard-scala.version>1.0.0-2</dropwizard-scala.version>
+    <dropwizard.test.version>1.0.0</dropwizard.test.version>
```

Comment:
Doesn't seem like this is used, and assuming this is the version of the dropwizard testing pom to use it's probably already being specified in the parent pom.